package.json

```diff
@@ -28,6 +28,7 @@
     "loader-utils": "~0.2.10",
     "node-sass": "^3.3.3",
     "object-assign": "~3.0.0",
+    "text-table": "^0.2.0",
```

Comment:
See: https://github.com/alleyinteractive/sasslint-webpack-plugin/issues/3

Appears the formatter file needs it, and somehow it didn't popup as extraneous for me. I think I missed it somehow originally (or it gets installed globally or something for some people).